interface/reports/patient_list_creation.php

```diff
@@ -35,34 +40,43 @@
     }
 }
 
-$search_options = array
-(
-    "Demographics"        => xl("Demographics"),
-    "Problems"            => xl("Problems"),
-    "Medications"         => xl("Medications"),
-    "Allergies"           => xl("Allergies"),
-    "Lab results"         => xl("Lab Results"),
-    "Communication"       => xl("Communication"),
-    "Insurance Companies" => xl("Insurance Companies")
-);
-
-$comarr = array
-(
+$search_options = [
+    "Demographics",
+    "Allergies",
+    "Problems",
+    "Medications",
+    "Prescriptions",
+    "Communication",
+    "Insurance Companies",
+    "Encounters",
+    "Observations",
+    "Procedures",
+    "Lab Results"
+];
```

Comment:
Rec not removing the translations. I see you are instead translating it in code below via the xl. Two reasons to not do this:
1. The translation pipeline that picks up translations that need to be translated in the codebase finds them in the xl() family functions, so best to translate the actual constant than a variable if possible.
2. Best to always keep the value and the label separate. Then the label (and associated translations) can be easily changed without messing up the values.